src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -117,5 +117,41 @@ public override Task ShutdownAsync(CancellationToken cancellationToken)
         {
             return Task.CompletedTask;
         }
+
+        private void ProcessTagObject(KeyValuePair<string, object> tag)
```

Comment:
Can we simply this so we don't have to maintain/add code in the future?

For example, we can probably remove this entire function and put the code directly in the foreach loop:

```csharp
foreach (var tag in activity.TagObjects)
{
    var array = tag as Array;
    if (array != null)
    {
        foreach (var item in array)
        {
            Console.WriteLine("item");
        }
    }
    else
    {
        Console.WriteLine("primitive type");
    }
}
```

Reply:
i see. this was the way it was before, but since it was going to increase the number of lines in the method i prefered to move to a method. but, sure, we can move inside.